djangocms_navigation/admin.py

```diff
@@ -23,6 +28,10 @@
 from .views import ContentObjectSelect2View, MenuContentPreviewView
 
 
+ROOT_MESSAGE = _(
```

Comment:
This is never done anywhere else in the packages, the messages are normally added inline unless they are reused or optional as per LOCK_MESSAGE.

This constant can be deleted and the error made inline.